### PR TITLE
Moar hobbies

### PR DESCRIPTION
**Pull request by ghost**
#### Summary
None

#### Purpose of change
AccountAlias can't be the only hobbyist, and we need more.

#### Describe the solution

Add more hobbies.
Remove *Hobby* Handloader, Roller Derby Player, Golfer, Boxer, & Game Master as those are hobbies.
Hobbyize XXXL/XS traits, and make them unselectable as traits.

#### Describe alternatives you've considered

Keeping in the sports professions as professional(🥁) athletes. Obsoleting Fisher as well.

#### Testing

TBD.

#### Additional context

AccountAlias was faster on having Archery & Skeet Shooting 😿.

### CODE REVIEW

File: data/json/hobbies.json [hunk 1]
```diff
@@ -91,5 +91,101 @@
...
+    "skills": [ { "level": 3, "name": "fabrication" }, { "level": 2, "name": "gun" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "game_mastering",
+    "name": "Game Master",
+    "description": "You've guided players' journeys through your dungeons and fortresses.  Now, you're last member of your party.",
+    "points": 2,
+    "traits": [ "PROF_DICMASTER" ],
```
Comment (actual-nh) [inline data/json/hobbies.json:122]:
```suggestion
    "traits": [ "PROF_DICEMASTER" ],
```
Pointed out by anothersimulacrum on Discord.

File: data/json/hobbies.json [hunk 2]
```diff
@@ -91,5 +91,101 @@
...
+    "points": 2,
+    "proficiencies": [ "prof_handloading" ],
+    "skills": [ { "level": 3, "name": "fabrication" }, { "level": 2, "name": "gun" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "game_mastering",
+    "name": "Game Master",
+    "description": "You've guided players' journeys through your dungeons and fortresses.  Now, you're last member of your party.",
```
Comment (actual-nh) [inline data/json/hobbies.json:120]:
```suggestion
    "description": "You've guided players' journeys through your dungeons and fortresses.  Now, you're the last member of your party.",
```

**Review by actual-nh — COMMENTED** [review]
Excellent start!

File: data/json/hobbies.json [hunk 3]
```diff
@@ -454,5 +454,183 @@
...
+    "description": "You could have been the next Browning, if not for the those pesky corpses walking around.  It's time to test for accuracy.",
+    "points": 2,
+    "proficiencies": [ "prof_gunsmithing_basic", "prof_gunsmithing_improv" ],
+    "skills": [ { "level": 2, "name": "fabrication" }, { "level": 1, "name": "gun" }, { "level": 1, "name": "traps" } ]
+  },
+  {
+    "subtype": "hobby",
+    "type": "profession",
+    "id": "roller_derby",
+    "description": "Being able your way through the pack is always useful, whether it's living or rotting.",
```
Comment (xanderrootslayer) [inline data/json/hobbies.json:549]: "Being able your way through the pack"?

**Pull request merged**